cloudwatch_metrics_common/include/cloudwatch_metrics_common/utils/metric_object.h

```diff
@@ -129,6 +121,14 @@ static MetricDatum metricObjectToDatum(const MetricObject &metrics, const int64_
     datum.SetStatisticValues(std::move(stats));
   }
 
+  auto mapped_unit = units_mapper.find(metrics.unit);
+  if (units_mapper.end() != mapped_unit) {
+    datum.SetUnit(mapped_unit->second);
```

Comment:
Default unit is seconds?

Reply:
(This isn't my code.)

This doesn't mean `seconds`. It is looking up the key in the `units_mapper` hash table (defined in the same header file) and `mapped_unit->second` is the value corresponding to that key.

Reply:
![image](https://user-images.githubusercontent.com/44818066/73498953-9d71c380-4373-11ea-943f-f861d4dc25b0.png)

Please create an issue for this! Thanks for clarifying.